src/pages/SamplePage/SamplePage.tsx

```diff
@@ -131,7 +202,7 @@ const SamplePage: React.FC = () => {
               slot={modalSlot}
               validationResult={
                 modalSlot
-                  ? validationResults?.[parseInt(sampleIndex)]?.[modalSlot.name]
+                  ? validationResults?.[sampleIndexInt]?.[modalSlot.name]
```

Comment:
Note to self: 
- Here are the docs about "optional chaining":  
  - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining